web/react/components/create_comment.jsx

```diff
@@ -109,6 +109,7 @@ export default class CreateComment extends React.Component {
         post.pending_post_id = `${userId}:${time}`;
         post.user_id = userId;
         post.create_at = time;
+        post.attachments = [];
```

Comment:
Argh this is stuff from another feature that shouldn't have been in here, sorry :-/